### PR TITLE
WIP added audio support, playing sound on eat and death

### PR DESCRIPTION
**Pull request by CoenSchutte**
Application crashes when running from /out folder, because of hard coded audio file locations

### CODE REVIEW

**Comment by CoenSchutte** [comment]
@Bowero Do you know how to specify a "dynamic" file path?

**Comment by Bowero** [comment]
The sound effects halt the game for me. That's something to look into. The IntelliJ bug seems to be IntelliJ-specific. Everything works fine here.

File: src/PlaySound.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import java.io.IOException;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.DataLine;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.SourceDataLine;
+
+public class PlaySound {
```
Comment (Bowero) [inline src/PlaySound.java]: This file is very big and complicated. There sure must be an easier way. How about [this](https://www.cs.cmu.edu/~illah/CLASSDOCS/javasound.pdf)?

Comment (CoenSchutte) [reply src/PlaySound.java]: Done!

**Comment by Bowero** [comment]
Are you happy with this PR now?

**Comment by CoenSchutte** [comment]
Yes! I believe we can merge this now

**Pull request merged**